source/_reference-t-z.html.md.erb

```diff
@@ -156,11 +252,13 @@ Refreshes an access token, as part of the OAuth process. For more information, s
 
      |     |
 -----|-----|
+<%= HEADERS %> | **Content-Type**: application/x-www-form-urlencoded
 <%= PARAMETERS %> | **grant_type** (required) -- must be set to "refresh_token"
  | **client_id** (required) -- client id for your app
  | **refresh_token** (required) -- refresh_token value that came with the access token
  | **redirect_uri** (optional) -- redirect URL registered for your app, including the protocol (for example, "https://")
```

Comment:
You mention that this is optional here, but earlier redirect_uri is listed as deprecated.

Reply:
Fixed.

Reply:
FYI: technically, it's just optional. But we're trying to de-emphasize it because it confuses people and it has to match exactly whatever callback URL they used when they registered their app or else it blows up.